CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Better pagination and number of variants returned by queries in `Search SNVs and INDELs` page
 - Refactored and simplified code used for collecting gene variants for `Search SNVs and INDELs` page
 - Fix sidebar panel icons in Case view
+- Ensure genome builds are input to Scout as strings from config files and CLI input
```

Comment:
Mmm looks like this line belongs to another PR?

Reply:
Fixed